etc/init.d/zfs.in

```diff
@@ -0,0 +1,74 @@
+# ZoL userland configuration.
+
+# Run `zfs mount -a` during system start?
+ZFS_MOUNT='yes'
+
+# Run `zfs unmount -a` during system stop?
+ZFS_UNMOUNT='yes'
+
+# Run `zfs share -a` during system start?
+# nb: The shareiscsi, sharenfs, and sharesmb dataset properties.
+ZFS_SHARE='yes'
```

Comment:
> Well, if we talk about the debs I provide, that was changed a very, very
> long time ago.
>

Ok. Maybe this is where we're misunderstanding each other. Because I just
did an upgrade today on a system with wheezy-daily and I was prompted to
resolve conflicts in /etc/default/zfs. I did the diffs, and saw these
changes. I then noticed the lock file name was changed to the new lock file
name used in this patch set. So I assumed that you were including this PR
in your builds (as you have done in the past). If I am in error, please let
me know.

If you can't handle the fact that others have different opinion on this
> than you and you try to _force_ your opinion, maybe you shouldn't comment?


Frans. Seriously. I am not trying to force my opinion on anyone. And I am
sorry that you continue to see it that way.


> You have iterated the exact same arguments the whole time, without
> supplying me with others when I have refuted yours. I do not agree with
> you, I've said that from the start. But since this isn't really relevant
> for this PR, but for the debs I provide, this discussion should be moved to
> the pkg-zfs repository tracker.
>

Ok. Then please respond to the issue I have already opened there
(zfsonlinux/pkg-zfs#153), asking for clarification about the wheezy-daily
builds.

---Alex